Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Spawnable/EntityIdPathMapperInterface.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/IO/Path/Path.h>
+#include <AzCore/RTTI/RTTI.h>
+
+namespace AzToolsFramework::Prefab::PrefabConversionUtils
+{
+    class Instance;
+
+    class EntityIdPathMapperInterface
+    {
+    public:
+        AZ_RTTI(EntityIdPathMapperInterface, "{1D1E032C-EEBA-449B-A265-03A8DE937EC9}");
+
+        virtual AZ::IO::PathView GetHashedPathUsedForEntityIdGeneration(const AZ::EntityId) = 0;
+        virtual void SetHashedPathUsedForEntityIdGeneration(const AZ::EntityId, AZ::IO::PathView) = 0;
```

Comment:
Same question here, what is the path to? And why is it involved in entity id generation? A comment here would be great.

Reply:
I'll investigate! Looping in @AMZN-koppersr @srikappa-amzn @michabr as this is their domain.

Reply:
Same as other comment. Once the comment is added next to the member variable of the map, then this would become obvious.